consensus/tests/common.rs

```diff
@@ -200,7 +200,9 @@ mod tests {
 
     #[test]
     #[wasm_bindgen_test]
-    fn test_longer_chain_with_diff_blockchain_length() {
+    //Current chain: Chain A (Long Chain)
```

Comment:
nit: space after `//`

Reply:
sure i'll take care of this at other places too